### DISCUSSION TITLE
This EXE was marked as an Trojan, today.

### DISCUSSION THREAD

**Discussion by XR1MAN** [ChrisTitusTech/winutil#1746] [General]
![image](https://github.com/ChrisTitusTech/winutil/assets/77406986/11315bbc-5c59-4f4b-82a0-870352193438)

**Comment by DarkAoRaidenX** [comment]
Just bought the EXE and windows is flagging as a Trojan still Phonzy.A!ml and uploaded to VirusTotal and 14 flagged it.
EDIT: I tried adding it to exclude it and it wont let me lul
![05 04 2024 13 35 49](https://github.com/ChrisTitusTech/winutil/assets/101914106/3fd976ef-a30b-41f3-8dd4-1251d452c741)
![05 04 2024 13 38 48](https://github.com/ChrisTitusTech/winutil/assets/101914106/c8057c51-ea07-41c6-a1c5-039c63a42a1a)
![05 04 2024 13 52 34](https://github.com/ChrisTitusTech/winutil/assets/101914106/1e7d7660-a448-483a-b3cc-440a022de2f4)

**Reply by MichelPostma** [reply]
I've had it flag the same file when I purchased WinUtil
For me Windows Defender noted: TrojanDownloader;Powershell/Ploprolo.gen!
Don't really know what to make of this, especially with those virus total reports
What's going on here?

**Comment by ChrisTitusTech** [comment]
@XR1MAN @MichelPostma @DarkAoRaidenX Here is a breakdown I did on stream with how I create the EXE, code signing, and why the flags happen. 
Its an issue, but with how the AV works in today's world it is not an issue I can resolve easily. https://www.youtube.com/watch?v=LLGMWqi0hnE
